src/components/MobileControls/MobileControl/MobileControl.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
/* eslint-disable @typescript-eslint/no-explicit-any */ ?

Reply:
The onChangeValue and onItemClick functions accept the "value" argument, which is of any type, so I used any. 
But now I have replaced 
/* eslint-disable @typescript-eslint/no-explicit-any */
with 
// eslint-disable-next-line @typescript-eslint/no-explicit-any